l337-postgres/Cargo.toml

```diff
@@ -3,9 +3,11 @@ name = "l337-postgres"
 version = "0.3.0"
 authors = ["Jonathon Sheffield <samsmug@gmail.com>"]
 edition = "2018"
+license = "MIT OR Apache-2.0"
+description = "l3-37 manager for tokio-postgres"
```

Comment:
We changed the package name to l337 instead of l3-37 -> mainly to remove the need to think about l3-37 vs l3_37 within the code.